README.md

```diff
@@ -157,19 +157,23 @@ rules:
 
 You may use the following shortcut or assemble your own config using the granular settings described below.
 
-Make sure you have installed [`@typescript-eslint/parser`] which is used in the following configuration. Unfortunately NPM does not allow to list optional peer dependencies.
+Make sure you have installed [`@typescript-eslint/parser`] and [`eslint-import-resolver-typescript`] which are used in the following configuration. Unfortunately NPM does not allow to list optional peer dependencies.
 
 ```yaml
 extends:
   - eslint:recommended
   - plugin:import/recommended
-  - plugin:import/typescript # this line does the trick
+# the following lines do the trick
+  - plugin:import/typescript
+settings:
+  import/resolver:
+    # You will also need to install and configure the TypeScript resolver
+    # See also https://github.com/import-js/eslint-import-resolver-typescript#configuration
+    typescript: true
```

Comment:
```suggestion
    typescript: true
    node: true
```
even though the TS resolver covers all the things the node resolver does, it's still better for both to be listed